README.md

```diff
@@ -10,17 +10,28 @@ This module requires the following modules/libraries:
 
 * [Islandora](https://github.com/islandora/islandora)
 * [Tuque](https://github.com/islandora/tuque)
+* [jsTree](https://github.com/vakata/jstree) version 3.0.0. (Does not work with later versions.)
 
 ## Installation
 
 Install as usual, see [this](https://drupal.org/documentation/install/modules-themes/modules-7) for further information.
 
-Download/clone the [jsTree](https://github.com/vakata/jstree) library to `sites/all/libraries/jstree`. The module has been tested with the 3.0.0 release of jsTree.
+Download/clone the [jsTree](https://github.com/vakata/jstree) library to `sites/all/libraries/jstree`. Version 3.0.0 is required.
 
 ## Configuration
 
-* The "Islandora Serial Object Tree View" block may be enabled to view the full structure of a Serial object.
+* The "Islandora Serial Object Tree View" block may be enabled to view the full structure of a Serial object. Required to expose and manage serial object hierarchies.
 * The "View: Islandora Serial Object Intermediate PDF View" block may be enabled if the [Islandora Solr Views](https://github.com/Islandora/islandora_solr_views) module is installed.
+* To make PDFs visible when viewing serial objects, configure the solution pack (admin/islandora/solution_pack_config/serial) and check "Display PDFs and sections in the jsTree view".
+
+## Preparing and ingesting serial objects
+
+* To create a top-level container object (e.g. a journal), ingest an object with the Serial Object content model.
+* Under Serial Level Configuration, set up the structure of your serial. (e.g. Level 1 might use the MODS detail type "volume", and the label "Volume". Level 2 might be "issue" and "Issue".)
+* After clicking Next, the "Create Intermediate Children" screen comes up. Here you can add levels (e.g. volumes and issues), which will act as containers for your PDFs. 
+** Click "Add Another" to create a Level 1 container. It is automatically assigned the Level 1 label. 
```

Comment:
I don't think this is rendering correctly. I think Github MD uses white space for nested lists and not incrementing bullets. @bondjimbond  do you think you can clean that up?